services/api/handlers/projects.js

```diff
@@ -254,6 +254,27 @@ exports.patch = {
   }
 };
 
+exports.delShallow = function(request, reply) {
+  request.server.methods.projects.remove(
+    [
+      request.params.project
+    ],
+    function(err, result) {
+      if(err) {
```

Comment:
to test this wouldn't i just stub `remove` again though? when i do it i get an error about "re-wrapping' `remove`, which frankly i didn't fully understand, heh.